app/assets/stylesheets/views/_search.scss

```diff
@@ -133,6 +133,24 @@ main.search {
         float: left;
         width: $two-thirds;
       }
+
+      mark {
+        background-color: rgba(255, 255, 140, 0.5);
```

Comment:
Doesn't this colour already have an inferred meaning, ie something that's focused? If my search term matches the title of the search result – which doesn't seem too far fetched, then it'd look like the title was focused and I could hit space or enter to visit that result.

Just my reckons.